crates/ark/src/modules/positron/package.R

```diff
@@ -21,6 +21,32 @@
 #' @export
 .ps.rpc.is_installed <- .ps.is_installed
 
+# Returns a list containing:
+#   * the version string if the package is installed and NULL otherwise
+#   * a minimum version to check for, with '0.0.0' as a dummy default
+#   * a logical indicating if package is installed at or above the minimum version
+#  This may seem weird, but it's impractical for positron-r to do version
+#  comparisons.
+#' @export
+.ps.rpc.packageVersion <- function(pkg, checkAgainst = "0.0.0") {
+    installed <- system.file(package = pkg) != ""
+
+    if (installed) {
+        version <- utils::packageVersion(pkg)
+        list(
+          version = as.character(version),
+          checkAgainst = as.character(checkAgainst),
+          checkResult = version >= checkAgainst
```

Comment:
If you move to `minimumVersion` and drop `checkAgainst` from the returned value, it could just be

```r
list(version = version/NULL, compatible = TRUE/FALSE)
```

So you end up with 3 states

```r
# Not installed (caller knows package name, for error reporting purposes)
list(version = NULL, compatible = FALSE)

# Not compatible (caller knows minimum version, for error reporting purposes)
list(version = version, compatible = FALSE)

# Good to go
list(version = version, compatible = TRUE)
```

Reply:
Well, I'm trying to make argument names and shape match what's on the other side in positron-r. But I don't feel strongly and can just code around whatever form y'all want here.

Reply:
On quick glance it looks like `minimumVersion` and `compatible` would also be nice and expressive in https://github.com/posit-dev/positron/pull/5365/files too

In particular I find `minimumVersion` much easier to understand on first glance than `checkAgainst`

Reply:
Yeah I'm changing it, just haven't pushed yet. Working on both ends.